common/check_oracle

```diff
@@ -64,12 +64,10 @@ ORDER BY inst_id, mname.metric_name, mname.metric_unit'''
         for (metric, value) in cursor:
             m = 'oracle.%s' % metric.replace(' ', '_')
             print(m, value, ' ')
-            monitor.record_gauge(stat=m, value=value)
+            probe_metrics.gauge(name=m).set(value)
```

Comment:
This doesn’t work because `Invalid metric name: rucio_probes_oracle_I/O_Megabytes_per_Second`. I think this might warrant a closer look at other places where the metric name is not hardcoded.

Reply:
OK, the new line 67 takes care of that

Reply:
Forgive me, what am I missing?

Reply:
I meant line 67 in the new commit, so now it goes like this:
```
            safe_metric = metric.replace(' ', '_')  # Convert space to underscore
            re.sub(r'\W+', '', safe_metric)  # Remove any other odd characters
            m = 'oracle.%s' % safe_metric
            ```

Reply:
Please note that `re.sub()` does not modify in place.

Reply:
That's what I get for relying on StackOverflow.

Fixed.